js/client-teambuilder.js

```diff
@@ -1144,10 +1146,20 @@
 		},
 		validate: function () {
 			var format = this.curTeam.format;
+			var prerequisiteErrors = [];
 			if (!format) {
-				app.addPopupMessage('You need to pick a format to validate your team.');
+				prerequisiteErrors.push('- You need to pick a format to validate your team.');
+			}
```

Comment:
Hmm it seems like this case will never happen according to the current logic (I just added to what was already there). I think this can be removed?